en/docs/Administer/ProductSecurity/enabling-hostname-verification.md

```diff
@@ -1,97 +1,41 @@
-
-
-If you are using a WSO2 product that is based on a Carbon version between 4.4.6 and 4.4.16,  hostname verification is **disabled** by default. This setting is disabled using the `org.wso2.ignoreHostnameVerification` property in the product startup script ( `wso2server.sh` for Linux, or `wso2server.bat` for Windows) that is stored in the `<PRODUCT_HOME>/bin/` directory.
-
-``` java
-    -Dorg.wso2.ignoreHostnameVerification=true
-```
-
-### Configuring hostname verification (Carbon 4.4.17 onwards)
-
-!!! info
-Before you begin, be sure to check the [prerequisites](#EnablingHostNameVerification-Prerequisites) .
-
-
-If you are using a WSO2 product that is based on Carbon 4.4.17 or a later version, hostname verification is **enabled** by default. This is done using the `httpclien` t `.hostnameVerifier` property in the product's startup script ( `wso2server.sh` for Linux and `wso2server.bat` for Windows) as shown below. The product startup script is stored in the `<PRODUCT_HOME>/bin` directory. This property will be effective during server startup.
-
-``` java
-    -Dhttpclient.hostnameVerifier="DefaultAndLocalhost"
-```
-
-The values you can use with this property are explained below. Note that these values will behave the same as synapse hostname verification options.
-
--   **DefaultAndLocalhost:** This is the value that is enabled, by default. This means that all hostnames, except the ones listed below, will be verified against the hostnames specified in the product's SSL certificate. That is, the following hostnames will be allowed regardless of the server's certificate.
-
-    -   localhost
-
-    -   localhost.localdomain
-
-    -   127.0.0.1
-
-    -   ::1
-
-    Note that if the wildcard symbol is used to specify a hostname in the SSL certificate (such as \*. [foo.com](http://foo.com/) ), all the subdomains of \*. [foo.com](http://foo.com/) are also included. That is, a hostname that matches a subdomain of \*. [foo.com](http://foo.com/) will also be allowed access.
-
--   **Strict:** When this mode is enabled, hostnames will be strictly verified against the hostname specified in the product's SSL certificate. For example, if "\*. [foo.com](http://foo.com/) " is specified as the hostname in the certificate, only the hostnames at the same level will be authorized by the server. That is, subdomains such as " [a.b.foo.com](http://a.b.foo.com/) " will **not** be authorized.
-
--   **AllowAll:** This option turns off hostname verification for the server. Note that this is not recommended in a production setup and should only be used for demonstrations and testing.
-
-        !!! note
     **Important!**
+    
+    Be sure to enable hostname verification for your production environment. This is necessary to avoid security issues in production. See the full list of [security recommendations for production](https://docs.wso2.com/display/ADMIN44x/Security+Guidelines+for+Production+Deployment) environments.
```

Comment:
Dont point to the Admin guide. The admin guide content was moved into APIM, so update the link pls.